javascript/message.js

```diff
@@ -0,0 +1,15 @@
+document.observe("dom:loaded", function() {
+    Event.observe('message-send', 'click', function(event) {
```

Comment:
Would this be better as `'submit'` rather than `'click'`? That way keyboard shortcuts for submitting forms would still work.

Reply:
Maybe? I didn't want to have to deal with preventing the default form behavior in that case.

Reply:
I will try to do some more experimenting with that

Reply:
I'm having trouble making it work as well with 'submit'. Some older browsers try to keep the default behavior, and end up submitting the form twice.

Reply:
Are there keyboard shortcuts that currently work?

Reply:
https://stackoverflow.com/questions/15612987/stopping-form-submit-with-prototype-js

https://stackoverflow.com/questions/8291805/stop-event-in-ie-9-without-upgrading-to-prototype-1-7

Looks like this is an interaction between earlier versions of Prototype and older browsers. Upgrading prototype could fix it (I don't know if it's backwards compatible, though) - or we could change the way the form is invoked generally.

[Alt] + [Shift] + [S] will submit forms in most browsers on windows, I believe. I have a mac, so can't test it (I don't believe there's an equivalent shortcut by default on mac).

Reply:
In my humble opinion we should leave it listening to the click event for this change. Perhaps upgrading prototype is a good idea anyway, but I leave that up to you and others who know more than me about the code base as a whole.

[Alt] + [Shift] + [S] didn't seem to work for me.

Reply:
I opened #331 to remind us to upgrade prototype, and I'll accept this change. When we apply it to the server we'll have to be careful in case it breaks message sending for anyone.

Thanks very much for this!!